### PR TITLE
added tests for logging and topic handler

### PR DESCRIPTION
**Pull request by p2h5**

### CODE REVIEW

File: libs/kafka/topichandler.py [hunk 1]
```diff
@@ -15,10 +15,11 @@ def create_topic_if_not_exists(kafkaserver, topicname):
...
-            admin_client.create_topics(new_topics=topic_list, validate_only=False)
-            print("[+] Created a new topic. Name: {}".format(topicname))
+        if topicname is not None:
+            if (topics := consumer.topics()) is not None and not topicname in topics:
+                admin_client = KafkaAdminClient(bootstrap_servers=kafkaserver, client_id='lib', api_version=(2, 7, 0))
+                topic_list = [NewTopic(name=topicname, num_partitions=1, replication_factor=1)]
+                admin_client.create_topics(new_topics=topic_list, validate_only=False)
+                print("[+] Created a new topic. Name: {}".format(topicname))
     except Exception as error:
         print(error)
```
Comment (DerAlexmeister) [inline libs/kafka/topichandler.py:25]: Logging missing?

**Review by DerAlexmeister — CHANGES_REQUESTED** [review]

File: libs/kafka/topichandler.py (hunk 1 above)
Comment (p2h5) [reply libs/kafka/topichandler.py:25]: Yes. I was not sure, if it is missing for a particular reason.
If not, I could add it.

Comment (DerAlexmeister) [reply libs/kafka/topichandler.py:25]: Would be great. I missed it.

Comment (p2h5) [reply libs/kafka/topichandler.py]: Fixed.

Comment (DerAlexmeister) [reply libs/kafka/topichandler.py]: Tests do fail.

Comment (p2h5) [reply libs/kafka/topichandler.py:26]: I think the logging caused the circular dependency. I removed it for now. The tests should be working again.

Comment (DerAlexmeister) [reply libs/kafka/topichandler.py:26]: Merge?

Comment (p2h5) [reply libs/kafka/topichandler.py:26]: From my side ok.

**Pull request merged**